src/main/java/de/uhd/ifi/se/decision/management/jira/webhook/WebhookEventListener.java

```diff
@@ -0,0 +1,70 @@
+package de.uhd.ifi.se.decision.management.jira.webhook;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import com.atlassian.event.api.EventListener;
+import com.atlassian.event.api.EventPublisher;
+import com.atlassian.jira.event.issue.IssueEvent;
+import com.atlassian.jira.event.issue.link.IssueLinkCreatedEvent;
+import com.atlassian.jira.event.issue.link.IssueLinkDeletedEvent;
+import com.atlassian.jira.event.type.EventType;
+import com.atlassian.plugin.spring.scanner.annotation.imports.JiraImport;
+
+import de.uhd.ifi.se.decision.management.jira.model.DecisionKnowledgeElement;
+import de.uhd.ifi.se.decision.management.jira.model.DecisionKnowledgeElementImpl;
+import de.uhd.ifi.se.decision.management.jira.persistence.ConfigPersistence;
+
+/**
+ * Triggers the webhook when JIRA issues are created, updated, or deleted or
+ * when links between JIRA issues are created or deleted
+ */
+@Component
+public class WebhookEventListener {
+
+	@Autowired
+	public WebhookEventListener(@JiraImport EventPublisher eventPublisher) {
+		eventPublisher.register(this);
+	}
+
+	@EventListener
+	public void onIssueEvent(IssueEvent issueEvent) {
+		String projectKey = issueEvent.getProject().getKey();
+		if (ConfigPersistence.isWebhookEnabled(projectKey)) {
+			Long eventTypeId = issueEvent.getEventTypeId();
+			DecisionKnowledgeElement decisionKnowledgeElement = new DecisionKnowledgeElementImpl(issueEvent.getIssue());
+			System.out.println("event");
+			if (eventTypeId.equals(EventType.ISSUE_CREATED_ID) || eventTypeId.equals(EventType.ISSUE_UPDATED_ID)) {
+				WebhookConnector connector = new WebhookConnector(projectKey);
+				connector.sendElementChanges(decisionKnowledgeElement);
```

Comment:
Is it possible in Java to use just one line like this:
`(new WebhookConnector(projectKey)).connector.sendElementChanges(decisionKnowledgeElement);`
? If yes, why not use it?